app/assets/stylesheets/admin/rebate_form.css

```diff
@@ -229,8 +229,16 @@
 
```

Comment:
suggest

```
.rebate-qrcode {
  margin: 5em auto;
  background-color: black;
  width: 20%;
  min-width: 350px;
  max-width: 500px;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}
```

Reply:
:sparkles:  I've had some time to investigate this a bit further this morning. Now that I understand the constraints of this issue, I think the general approach you've gone with is the right one. However a good thing to do when you end up with CSS like this is to try removing each line in turn to make sure they are all active. In this case I found that neither `display: flex` or `margin-right` are having an impact so lets remove those and then git this merge!!! :sparkles: